spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  describe 'date formatting constants' do
+    context 'FULL_DATE_FORMAT_STRING' do
+      it 'exists' do
+        expect(ApplicationHelper::FULL_DATE_FORMAT_STRING).to be_present
+      end
+
+      it 'is a String' do
+        expect(ApplicationHelper::FULL_DATE_FORMAT_STRING).to be_a_kind_of String
+      end
+    end
+
+    context 'MONTH_YEAR_DATE_FORMAT_STRING' do
+      it 'exists' do
+        expect(ApplicationHelper::MONTH_YEAR_DATE_FORMAT_STRING).to be_present
+      end
+
+      it 'is a String' do
+        expect(ApplicationHelper::MONTH_YEAR_DATE_FORMAT_STRING).to be_a_kind_of String
+      end
+    end
+  end
+
+  context ':format_full_date' do
+    it 'formats a date' do
```

Comment:
Constants are now tested that they are set properly, at the expense of having a single authoritative place where date formats are set.  If `.strftime` changes a deprecation warning should be issued.